README.md

```diff
@@ -22,17 +22,11 @@ required.
 
 ## Compatibility
 
-This module only supports Smart Proxy 1.11 or higher as of version 3.0, as the
+This module only supports Smart Proxy 1.12 or higher as of version 4.0, as the
 configuration layout changed significantly.
 
-To configure older versions of the Smart Proxy (1.5 to 1.10), use version 2.x
-of this module.
-
-### 1.11 compatibility notes
```

Comment:
After this being merged, the module is only compatible with 1.12+ and no parameters can be set to get backwards compatible config files, so I hope the existing notes are sufficient. ;)